sdk/src/main/java/io/radar/sdk/RadarHuaweiLocationClient.kt

```diff
@@ -0,0 +1,135 @@
+package io.radar.sdk
+
+import android.annotation.SuppressLint
+import android.app.PendingIntent
+import android.content.Context
+import android.location.Location
+import android.os.Looper
+import com.huawei.hms.location.*
+
+@SuppressLint("MissingPermission")
+internal class RadarHuaweiLocationClient(
+    context: Context,
+    private val logger: RadarLogger
+): RadarAbstractLocationClient() {
+
+    @SuppressLint("VisibleForTests")
+    val locationClient = FusedLocationProviderClient(context)
+    val geofenceService = GeofenceService(context)
```

Comment:
Since these lines are identical in the `RadarGoogleLocationClient`, can they be moved into the superclass?

Reply:
I think they're coming from a different package, so it wouldn't work `import com.huawei.hms.location.*`